mqtt/mqtt-edgehub/src/command.rs

```diff
@@ -75,11 +72,11 @@ impl CommandHandler {
     }
 
     pub async fn run(mut self) -> Result<(), CommandHandlerError> {
-        let subscribe_topics = &[TOPIC_FILTER.to_string()];
+        let subscribe_topic = &[DISCONNECT_TOPIC.to_string()];
 
-        if !self.subscribe(subscribe_topics).await? {
+        if !self.subscribe(subscribe_topic).await? {
```

Comment:
nit: why not return the error directly from subscribe method? `if` here is not really necessary.

Reply:
Oh yes I can return my own error. I will change.